cli/azd/pkg/output/colors.go

```diff
@@ -48,6 +49,21 @@ func WithBackticks(text string) string {
 }
 
 // WithHyperlink wraps text with the colored hyperlink format escape sequence.
-func WithHyperlink(url string, text string) string {
-	return WithLinkFormat(fmt.Sprintf("\033]8;;%s\007%s\033]8;;\007", url, text))
+// When url and displayName are specified the displayName is used as the link display name
+// When url is an empty string the url is used as the link and the display name
+func WithHyperlink(url string, displayName string) string {
+	if displayName == "" {
+		displayName = url
+	}
+
+	var noColor = os.Getenv("NO_COLOR") != ""
```

Comment:
More appropriate to use `color.NoColor` for completeness, which is determined from `NO_COLOR` and a series of other detection.

Reply:
Updated to check against `color.NoColor`.